uberpoet/filegen.py

```diff
@@ -55,7 +64,7 @@ def __init__(self, text, functions, classes):
 
 class FileGenerator(object):
 
-    def gen_file(self, class_count, function_count, line_goal):
```

Comment:
the last param was never used. Upon writing a test and using this base class FileGenerator it was crashing.